pyproject.toml

```diff
@@ -0,0 +1,28 @@
+[tool.poetry]
+name = "trackma"
+version = "2022.10"
+description = "Open multi-site list manager"
+authors = ["z411 <z411@krutt.org>"]
+license = "GPL-3.0-or-later"
+readme = "README.md"
+
+[tool.poetry.dependencies]
+python = "3.7.14"
+inotify = "0.2.10"
+pillow = "9.2.0"
+pycairo = "1.21.0"
+pydbus = "0.6.0"
+pygobject = "3.42.1"
+pyinotify = "0.9.6"
+pypresence = "4.2.1"
+pyqt5 = "5.15.7"
+requests = "2.28.1"
+requests-oauthlib = "1.3.1"
+twitter = "1.19.3"
+urwid = "2.1.2"
+setuptools = "63.1.0"
+sip = "6.6.2"
```

Comment:
The UI-specific dependencies should be optional and specified with extras.

See https://python-poetry.org/docs/pyproject/#extras

Reply:
Done